.github/actions/test-dotnet/Invoke-SolutionTests.ps1

```diff
@@ -56,8 +56,6 @@ foreach ($test in $tests)
 
     $dotnetTestSwitches = @(
         '--configuration', $Configuration
-        '--no-restore',
-        '--no-build',
         '--nologo',
         '--logger', 'trx;LogFileName=test-results.trx'
         # This is for xUnit ITestOutputHelper, see https://xunit.net/docs/capturing-output.
```

Comment:
Can't you add `$optOut` here? That way we would only need to use `@dotnetTestSwitches` down there.